routing/directions_engine.cpp

```diff
@@ -0,0 +1,83 @@
+#include "routing/directions_engine.hpp"
+
+#include "base/assert.hpp"
+
+namespace
+{
+double constexpr KMPH2MPS = 1000.0 / (60 * 60);
+}  // namespace
+
+namespace routing
+{
+void IDirectionsEngine::CalculateTimes(IRoadGraph const & graph, vector<Junction> const & path,
+                                       Route::TTimes & times) const
+{
+  times.clear();
+  if (path.size() <= 1)
+    return;
+
+  double const speedMPS = graph.GetMaxSpeedKMPH() * KMPH2MPS;
```

Comment:
Interesting... It looks like an old mistake. This code is used for time calculation for pedestrian routing.